app/models/project.rb

```diff
@@ -1,4 +1,18 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def challenge_theme
+    challenge.theme
```

Comment:
This is fine to encapsulate in a model method, but it's also okay to drill into a challenge theme from the view as well! IF you did `project.challenge.theme` in the view, that's totally okay. You're not manipulating data or doing calculations, just reading data